### PR TITLE
Dbz 3772 update oracle connector deployment instructions

### PR DESCRIPTION
**Pull request by roldanbob**
[DBZ-3772](https://issues.redhat.com/browse/DBZ-3772)

This change harmonizes the deployment instructions for the Oracle connector with the instructions that we provide for the other Debezium connectors, including conditionalizing steps for exclusive upstream or downstream use.

Tested in a local Antora build. Also tested that the annotations for splitting the file for downstream use return the expected results, and that the conditionalization behaves as expected.  

Please backport these changes to the 1.5 branch.

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/oracle.adoc [hunk 1]
```diff
@@ -1140,48 +1140,359 @@ sqlplus sys/top_secret@//localhost:1521/ORCLCDB as sysdba
...
-Copy `ojdbc8.jar` and `xstreams.jar` (not needed when using the LogMiner-based capture implementation) into Kafka Connect's _libs_ directory.
-Lastly, create an environment variable, `LD_LIBRARY_PATH`, that points to the instant client directory, as shown below:
+// Type: procedure
+// Title: Obtaining the Oracle JDBC driver
+// ModuleID: obtaining-the-oracle-jdbc-driver
+[id="obtaining-oracle-jdbc-driver-and-xstreams-api-files"]
+=== Obtaining the Oracle JDBC driver and XStream API files
 
+ifdef::community[]
+Depending on your Oracle environment, you must have either the Oracle JDBC driver (`ojdbc8.jar`) or XStream API (`xstreams.jar`) to run the {prodname} Oracle connector.
```
Comment (Naros) [inline documentation/modules/ROOT/pages/connectors/oracle.adoc:1194]: This sentence is a tad misleading.  The `ojdbc8.jar` is required regardless of which adapter the connector config uses.  The `xstreams.jar` is needed only if the connector config uses the `xstream` adapter.

**Review by Naros — CHANGES_REQUESTED** [review]
Overall this LGTM, just one nit-pick I found.

File: documentation/modules/ROOT/pages/connectors/oracle.adoc (hunk 1 above)
Comment (roldanbob) [reply documentation/modules/ROOT/pages/connectors/oracle.adoc:1194]: Ah, ok. Thanks, I (obviously) wasn't clear on that.

Comment (roldanbob) [reply documentation/modules/ROOT/pages/connectors/oracle.adoc]: OK, fixed.

**Comment by Naros** [comment]
Applied, thanks @roldanbob !

**Review by Naros — APPROVED** [review]

**Pull request merged**